Fix vllm build issue

yangw-dev opened the pull request:
Fixes #ISSUE_NUMBER
unstable https://github.com/pytorch/pytorch/issues/164362

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

huydhn commented on .github/ci_configs/vllm/Dockerfile.tmp_vllm:
```diff
@@ -338,20 +338,23 @@ ARG torch_cuda_arch_list='8.0;8.9;9.0a;10.0a;12.0'
+RUN uv pip freeze | grep -E 'setuptools|packaging|build'
 
 ENV TORCH_CUDA_ARCH_LIST=${torch_cuda_arch_list}
+ENV FLASHINFER_CUDA_ARCH_LIST=${torch_cuda_arch_list}
```
Btw, there is a new script now to build FlashInfer from vLLM https://github.com/vllm-project/vllm/pull/25782 called `tools/flashinfer-build.sh`.  You could try to see if that script works for us here (later)

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

huydhn committed:
Minor tweak

huydhn committed:
Remove somewhat unnecessary change

huydhn committed:
[no ci] Another tweak

huydhn committed:
Another tweak

huydhn committed:
Bring in https://github.com/vllm-project/vllm/pull/25730

huydhn committed:
Would this work?

Why did it link with CUDA runtime here https://github.com/pytorch/pytorch/actions/runs/18187580810/job/51775137099?pr=164361#step:17:20314?

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config

yangw-dev committed:
add config
